packages/image/src/index.vue

```diff
@@ -185,6 +185,8 @@ export default defineComponent({
       // so it can behave consistently
       Object.keys(attributes)
         .forEach(key => {
+          // avoid onload to be overwritten
+          if (key.toLowerCase() === 'onload') return
```

Comment:
What about we do something like this:
```ts
Object.keys(attributes)
        .forEach(key => {
          // avoid onload to be overwritten
          const value = attributes[key]
          img.setAttribute(key, value)
        })
const prevOnload = image.onload;
image.onload = (e) => {
   handleLoaded(e, img)
   prevOnload?.()
}
```

Reply:
The main idea behind this is that we need to hijack the event instead of prevent it from binding, sometimes user need to know the exact moment that image is loaded.